gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -660,3 +688,181 @@ def _get_variant_calls(self, variant_proto):
         call.info[k] = data
       calls.append(call)
     return calls
+
+
+def open_bgzf(file_name):
```

Comment:
I would think this function and class definition belong in a separate file but I am not too familiar with pythons rules on this sort of thing...